.drone.yml

```diff
@@ -92,7 +114,7 @@ steps:
   - name: destroy
     image: quay.io/sighup/e2e-testing-drone-plugin:v0.4.2
```

Comment:
I said to replace v0.4.2 -> v0.5.0 in all appearance of `quay.io/sighup/e2e-testing-drone-plugin:v0.4.2`.
Can you do it?